kafka-handler/README.md

```diff
@@ -25,6 +25,10 @@ If you want to switch serializer/deserializer classes you can use alter table.
 ```sql
 ALTER TABLE kafka_table SET TBLPROPERTIES ("kafka.serde.class"="org.apache.hadoop.hive.serde2.avro.AvroSerDe");
 ``` 
+
+If you use Confluent Avro serialzier/deserializer with schema registry you may want to remove 4 bytes from beginning that represents schema ID from registry. 
+It can be done with settings `"avro.serde.magic.bytes"="true"`. It's recommended to set `"avro.schema.url"="http://schemaregistry/SimpleDocument.avsc"`
```

Comment:
Note: `.avsc` isn't directly accessible from the Confluent Schema Registry. 

Link https://github.com/confluentinc/schema-registry/pull/686

Reply:
\+ I don't know if people really understand what "magic bytes" mean here... Would it make more sense to rename to something like `avro.serde.type="confluent"`? 

Or really, just a separate SerDe entirely?

Reply:
\+ Can `avro.schema.literal` be given too? Should that be documented?

Reply:
Yea, this is true the schema could be from s3 or other source. I put that `schemaregistry` hostname as a placeholder which is misleading.

I like that idea with `avro.serde.type="confluent"`

That `avro.schema.literal` is from 2014 and from what I read from code it has higher priority than `avro.schema.url`. Well, there is never enough documentation tho :)

Reply:
I've added 2 properties `avro.serde.type` and `avro.serde.skip.bytes` and if type is set to confluent it skip first 5 bytes. These 5 bytes can be set to different value with avro.serde.skip.bytes property.